### PR TITLE
Checking antagonists will now message admins

### PR DESCRIPTION
**Pull request by tkdrg**

### CODE REVIEW

**Comment by Jordie0608** [comment]
The last time this was tried for reference #12183

**Comment by Iamgoofball** [comment]
admins need to stop getting their panties in a twist over PRs threatening the conspiracy

:+1:

**Comment by The-Albinobigfoot** [comment]
:1: 
SHUT IT DOWN, THE GOY KNOW.

**Comment by Shadowlight213** [comment]
Should do the thing suggested last time and make it only show up if they do it while playing.

**Comment by MrStonedOne** [comment]
check antag contains a lot of other info then just the antags.

and it is used often, closed and re-opened, etc etc etc.

i don't know how many times i would close and re-open it to see the blob tile win count as admin ghost

or how many times i would close and re-open it to see if mulligen triggered properly.

Fuck off with this shit, make it exempt ghosts or bust.

**Comment by tkdrg** [comment]
sure whatever

**Comment by MrStonedOne** [comment]
Should make that a inverse observer check, otherwise admins can check antag in lobby then join the round with no admin messaging.

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -364,6 +364,8 @@ var/list/admin_verbs_hideable = list(
 	if(holder)
 		holder.check_antagonists()
 		log_admin("[key_name(usr)] checked antagonists.")	//for tsar~
+		if(isliving(usr))
```
Comment (MrStonedOne) [inline code/modules/admin/admin_verbs.dm]: Should make that a inverse observer check, other wise admins can check antags in lobby and late join without a message to admins.

**Comment by tkdrg** [comment]
Done

**Comment by YotaXP** [comment]
Can't the admin just aghost before bringing it up then?

**Comment by tkdrg** [comment]
Yep. But Aghost messages admins as well.

**Comment by Aloraydrel** [comment]
add a refresh button to it tbh

**Comment by tkdrg** [comment]
That will solve itself once we rewrite the entire admin interface using a pretty nanoUi with tabs and stuff.

**Comment by YotaXP** [comment]
> once we rewrite the entire admin interface using a pretty nanoUi with tabs and stuff.

**Comment by neersighted** [comment]
Someday...

**Pull request merged**